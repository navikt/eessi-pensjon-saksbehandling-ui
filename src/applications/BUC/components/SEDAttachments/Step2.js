import React, { useState, useEffect } from 'react'
import PT from 'prop-types'
import _ from 'lodash'
import classNames from 'classnames'
import PDFEditor from 'applications/PDF/components/PDFEditor/PDFEditor'
import { Ingress, Hovedknapp, Flatknapp } from 'components/ui/Nav'

const Step2 = (props) => {
  const { t, files, setFiles, setStep } = props

  const onForwardButtonClick = () => {
    setStep('generate')
  }

  const onBackButtonClick = () => {
    setStep('select')
  }

  return <div>
    <PDFEditor {...props} />
    <Hovedknapp
      id='a-buc-c-sedattachmnents__next-button-id'
      className='a-buc-c-sedattachmnents__next-button'
      onClick={onForwardButtonClick}>
      {t('ui:next')}
    </Hovedknapp>
    <Flatknapp
      id='a-buc-c-sedattachmnents__next-button-id'
      className='a-buc-c-sedattachmnents__next-button'
      onClick={onBackButtonClick}>
      {t('ui:back')}
    </Flatknapp>
  </div>
}

Step2.propTypes = {
  actions: PT.object,
  t: PT.func.isRequired,
  setFiles: PT.func.isRequired
}

export default Step2
