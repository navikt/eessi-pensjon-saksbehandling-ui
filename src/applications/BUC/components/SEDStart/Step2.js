import React, { useState } from 'react'
import { Flatknapp, Hovedknapp, Row } from 'components/ui/Nav'

const Step2 = (props) => {

  const { actions, t, loading, backStep } = props

  const onBackButtonClick = () => {
    backStep()
  }

  const onForwardButtonClick = () => {
    console.log('Forward')
  }

  const onCancelButtonClick = () => {
    actions.resetBuc()
    actions.setMode('list')
  }

  const allowedToForward = () => {
    return true
  }

  return <Row>
    <div className='col-md-12'>
      <Hovedknapp
        id='a-buc-c-sedstart__forward-button-id'
        className='a-buc-c-sedstart__forward-button'
        disabled={!allowedToForward()}
        spinner={loading.creatingSed}
        onClick={ onForwardButtonClick }>
        {t('buc:form-orderSED')}
      </Hovedknapp>
      <Flatknapp
        id='a-buc-c-sedstart__back-button-id'
        className='a-buc-c-sedstart__back-button'
        onClick={onBackButtonClick}>{t('ui:back')}
      </Flatknapp>
      <Flatknapp
        id='a-buc-c-sedstart__cancel-button-id'
        className='a-buc-c-sedstart__cancel-button'
        onClick={onCancelButtonClick}>{t('ui:cancel')}
      </Flatknapp>
    </div>
 </Row>

}

export default Step2
