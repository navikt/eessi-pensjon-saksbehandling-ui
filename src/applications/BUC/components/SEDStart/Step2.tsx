import SEDP4000 from 'applications/BUC/components/SEDP4000/SEDP4000'
import { Buc } from 'applications/BUC/declarations/buc.d'
import Ui from 'eessi-pensjon-ui'
import React from 'react'
import { AllowedLocaleString, T, Validation } from 'types.d'

export interface Step2Props {
  buc: Buc;
  t: T;
  locale: AllowedLocaleString;
  _sed: string;
  showButtons: boolean;
  setShowButtons: Function;
  validation: Validation;
  setValidation: Function;
}

const Step2 = (props: Step2Props) => {
  const { buc, t, _sed, showButtons, setShowButtons } = props

  return (
    <>
      <div className='col-md-12'>
        <Ui.Nav.Systemtittel>{t('buc:step-startSEDTitle', {
          buc: t(`buc:buc-${buc.type}`),
          sed: _sed || t('buc:form-newSed')
        })}
        </Ui.Nav.Systemtittel>
        <hr />
      </div>
      {_sed === 'P4000' ? (
        <>
          <div className='col-md-8'>
            <SEDP4000 showButtons={showButtons} setShowButtons={setShowButtons} {...props} />
          </div>
          <div className='col-md-4' />
        </>
      ) : null}
    </>
  )
}

export default Step2
