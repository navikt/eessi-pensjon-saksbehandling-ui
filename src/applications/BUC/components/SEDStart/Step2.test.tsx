import { mount, ReactWrapper } from 'enzyme'
import _ from 'lodash'
import React from 'react'
import sampleBucs from 'resources/tests/sampleBucs'
import Step2, { Step2Props } from './Step2'

jest.mock('applications/BUC/components/SEDP4000/SEDP4000', () => {
  return () => { return <div className='mock-sedp4000' /> }
})

describe('applications/BUC/components/SEDStart/AttachmentStep2', () => {
  let wrapper: ReactWrapper
  const mockBucs = _.keyBy(sampleBucs, 'caseId')
  const initialMockProps: Step2Props = {
    buc: mockBucs['195440'],
    locale: 'nb',
    _sed: 'P4000',
    t: jest.fn(t => t),
    showButtons: true,
    setShowButtons: jest.fn(),
    validation: {},
    setValidation: jest.fn()
  }

  beforeEach(() => {
    wrapper = mount(<Step2 {...initialMockProps} />)
  })

  afterEach(() => {
    wrapper.unmount()
  })

  it('Renders', () => {
    expect(wrapper.isEmptyRender()).toBeFalsy()
    expect(wrapper).toMatchSnapshot()
  })

  it('Has proper HTML structure', () => {
    expect(wrapper.exists('.mock-sedp4000')).toBeTruthy()
  })
})
