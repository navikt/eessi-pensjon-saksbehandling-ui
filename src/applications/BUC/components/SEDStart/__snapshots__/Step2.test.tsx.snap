// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`applications/BUC/components/SEDStart/AttachmentStep2 Renders 1`] = `
<Step2
  _sed="P4000"
  aktoerId="123"
  buc={
    Object {
      "aktoerId": "1000070418092",
      "caseId": "195440",
      "creator": Object {
        "country": "NO",
        "institution": "NO:NAVAT07",
        "name": "NAV ACCEPTANCE TEST 07",
      },
      "institusjon": Array [
        Object {
          "country": "NO",
          "institution": "NO:NAVAT07",
        },
        Object {
          "country": "SE",
          "institution": "SE:DEMO001",
        },
        Object {
          "country": "FI",
          "institution": "FI:DEMO002",
        },
        Object {
          "country": "GB",
          "institution": "GB:DEMO003",
        },
        Object {
          "country": "PT",
          "institution": "PT:DEMO004",
        },
        Object {
          "country": "FR",
          "institution": "FR:DEMO005",
        },
      ],
      "lastUpdate": 1571731814000,
      "sakType": "",
      "seds": Array [
        Object {
          "allowsAttachments": true,
          "attachments": Array [
            Object {
              "documentId": "cac9db2726d54f2c9b51d1562b7b0a79",
              "fileName": "balrog.png",
              "id": "58828e7ff57943779ab7b24c522b1a3f",
              "lastUpdate": Object {
                "chronology": Object {
                  "calendarType": "iso8601",
                  "id": "ISO",
                },
                "dayOfMonth": 20,
                "dayOfWeek": "MONDAY",
                "dayOfYear": 140,
                "era": "CE",
                "leapYear": false,
                "month": "MAY",
                "monthValue": 5,
                "year": 2019,
              },
              "medical": false,
              "mimeType": "image/png",
              "name": "balrog.png",
            },
          ],
          "creationDate": 1559117379454,
          "firstVersion": Object {
            "date": 1559117379454,
            "id": "1",
          },
          "id": "90149c52a98044b599c3bf5d48537782",
          "lastUpdate": 1559080800000,
          "lastVersion": Object {
            "date": 1559080800000,
            "id": "r",
          },
          "participants": Array [
            Object {
              "organisation": Object {
                "accessPoint": null,
                "acronym": "12348",
                "activeSince": 1518476400000,
                "address": Object {
                  "country": "NO",
                  "postalCode": null,
                  "region": null,
                  "street": null,
                  "town": null,
                },
                "assignedBUCs": null,
                "contactMethods": null,
                "countryCode": "NO",
                "id": "NO:DEMO002",
                "location": null,
                "name": "DEMO002",
                "registryNumber": null,
              },
              "role": "Sender",
              "selected": true,
            },
            Object {
              "organisation": Object {
                "accessPoint": null,
                "acronym": "12347",
                "activeSince": 1508450400000,
                "address": Object {
                  "country": "NO",
                  "postalCode": null,
                  "region": null,
                  "street": null,
                  "town": null,
                },
                "assignedBUCs": null,
                "contactMethods": null,
                "countryCode": "NO",
                "id": "NO:DEMO001",
                "location": null,
                "name": "DEMO001",
                "registryNumber": null,
              },
              "role": "Receiver",
              "selected": false,
            },
          ],
          "status": "received",
          "type": "P2000",
          "version": "5",
        },
        Object {
          "allowsAttachments": true,
          "attachments": Array [],
          "creationDate": 1559117444000,
          "firstVersion": Object {
            "date": 1559117444000,
            "id": "1",
          },
          "id": "ff98ea9256244b908e19d4439db5bad3",
          "lastUpdate": 1559080800000,
          "lastVersion": Object {
            "date": 1559080800000,
            "id": "4",
          },
          "parentDocumentId": "90149c52a98044b599c3bf5d48537782",
          "participants": Array [
            Object {
              "organisation": Object {
                "accessPoint": null,
                "acronym": "12348",
                "activeSince": 1518476400000,
                "address": Object {
                  "country": "NO",
                  "postalCode": null,
                  "region": null,
                  "street": null,
                  "town": null,
                },
                "assignedBUCs": null,
                "contactMethods": null,
                "countryCode": "NO",
                "id": "NO:DEMO002",
                "location": null,
                "name": "DEMO002",
                "registryNumber": null,
              },
              "role": "Sender",
              "selected": false,
            },
            Object {
              "organisation": Object {
                "accessPoint": null,
                "acronym": "12347",
                "activeSince": 1508450400000,
                "address": Object {
                  "country": "NO",
                  "postalCode": null,
                  "region": null,
                  "street": null,
                  "town": null,
                },
                "assignedBUCs": null,
                "contactMethods": null,
                "countryCode": "NO",
                "id": "NO:DEMO001",
                "location": null,
                "name": "DEMO001",
                "registryNumber": null,
              },
              "role": "Receiver",
              "selected": false,
            },
          ],
          "status": "empty",
          "type": "P6000",
          "version": "4",
        },
        Object {
          "allowsAttachments": true,
          "attachments": Array [],
          "creationDate": 1559117443000,
          "firstVersion": Object {
            "date": 1559117443000,
            "id": "1",
          },
          "id": "549bab7141c54cfcb14902e7e3b107b4",
          "lastUpdate": 1559080800000,
          "lastVersion": Object {
            "date": 1559080800000,
            "id": "4",
          },
          "participants": Array [
            Object {
              "organisation": Object {
                "accessPoint": null,
                "acronym": "12347",
                "activeSince": 1508450400000,
                "address": Object {
                  "country": "NO",
                  "postalCode": null,
                  "region": null,
                  "street": null,
                  "town": null,
                },
                "assignedBUCs": null,
                "contactMethods": null,
                "countryCode": "NO",
                "id": "NO:DEMO001",
                "location": null,
                "name": "DEMO001",
                "registryNumber": null,
              },
              "role": "Receiver",
              "selected": false,
            },
            Object {
              "organisation": Object {
                "accessPoint": null,
                "acronym": "12348",
                "activeSince": 1518476400000,
                "address": Object {
                  "country": "NO",
                  "postalCode": null,
                  "region": null,
                  "street": null,
                  "town": null,
                },
                "assignedBUCs": null,
                "contactMethods": null,
                "countryCode": "NO",
                "id": "NO:DEMO002",
                "location": null,
                "name": "DEMO002",
                "registryNumber": null,
              },
              "role": "Sender",
              "selected": false,
            },
          ],
          "status": "empty",
          "type": "P3000_NO",
          "version": "4",
        },
        Object {
          "allowsAttachments": true,
          "attachments": Array [],
          "creationDate": 1559117444000,
          "firstVersion": Object {
            "date": 1559117444000,
            "id": "1",
          },
          "id": "1911923c542746ada10f7c922b46a271",
          "lastUpdate": 1559080800000,
          "lastVersion": Object {
            "date": 1559080800000,
            "id": "3",
          },
          "participants": Array [
            Object {
              "organisation": Object {
                "accessPoint": null,
                "acronym": "12348",
                "activeSince": 1518476400000,
                "address": Object {
                  "country": "NO",
                  "postalCode": null,
                  "region": null,
                  "street": null,
                  "town": null,
                },
                "assignedBUCs": null,
                "contactMethods": null,
                "countryCode": "NO",
                "id": "NO:DEMO002",
                "location": null,
                "name": "DEMO002",
                "registryNumber": null,
              },
              "role": "Sender",
              "selected": false,
            },
            Object {
              "organisation": Object {
                "accessPoint": null,
                "acronym": "12347",
                "activeSince": 1508450400000,
                "address": Object {
                  "country": "NO",
                  "postalCode": null,
                  "region": null,
                  "street": null,
                  "town": null,
                },
                "assignedBUCs": null,
                "contactMethods": null,
                "countryCode": "NO",
                "id": "NO:DEMO001",
                "location": null,
                "name": "DEMO001",
                "registryNumber": null,
              },
              "role": "Receiver",
              "selected": false,
            },
          ],
          "status": "empty",
          "type": "P5000",
          "version": "3",
        },
        Object {
          "allowsAttachments": true,
          "attachments": Array [],
          "creationDate": 1559117443000,
          "firstVersion": Object {
            "date": 1559117443000,
            "id": "1",
          },
          "id": "9ebb27b848c644ba8a8ba95772403618",
          "lastUpdate": 1559080800000,
          "lastVersion": Object {
            "date": 1559080800000,
            "id": "4",
          },
          "participants": Array [
            Object {
              "organisation": Object {
                "accessPoint": null,
                "acronym": "12348",
                "activeSince": 1518476400000,
                "address": Object {
                  "country": "NO",
                  "postalCode": null,
                  "region": null,
                  "street": null,
                  "town": null,
                },
                "assignedBUCs": null,
                "contactMethods": null,
                "countryCode": "NO",
                "id": "NO:DEMO002",
                "location": null,
                "name": "DEMO002",
                "registryNumber": null,
              },
              "role": "Sender",
              "selected": false,
            },
            Object {
              "organisation": Object {
                "accessPoint": null,
                "acronym": "12347",
                "activeSince": 1508450400000,
                "address": Object {
                  "country": "NO",
                  "postalCode": null,
                  "region": null,
                  "street": null,
                  "town": null,
                },
                "assignedBUCs": null,
                "contactMethods": null,
                "countryCode": "NO",
                "id": "NO:DEMO001",
                "location": null,
                "name": "DEMO001",
                "registryNumber": null,
              },
              "role": "Receiver",
              "selected": false,
            },
          ],
          "status": "empty",
          "type": "X009",
          "version": "4",
        },
        Object {
          "allowsAttachments": true,
          "attachments": Array [],
          "creationDate": 1559117444000,
          "firstVersion": Object {
            "date": 1559117444000,
            "id": "1",
          },
          "id": "4ae14845e31d4d0db52acb692d9abe8e",
          "lastUpdate": 1559080800000,
          "lastVersion": Object {
            "date": 1559080800000,
            "id": "2",
          },
          "participants": Array [
            Object {
              "organisation": Object {
                "accessPoint": null,
                "acronym": "12348",
                "activeSince": 1518476400000,
                "address": Object {
                  "country": "NO",
                  "postalCode": null,
                  "region": null,
                  "street": null,
                  "town": null,
                },
                "assignedBUCs": null,
                "contactMethods": null,
                "countryCode": "NO",
                "id": "NO:DEMO002",
                "location": null,
                "name": "DEMO002",
                "registryNumber": null,
              },
              "role": "Sender",
              "selected": false,
            },
            Object {
              "organisation": Object {
                "accessPoint": null,
                "acronym": "12347",
                "activeSince": 1508450400000,
                "address": Object {
                  "country": "NO",
                  "postalCode": null,
                  "region": null,
                  "street": null,
                  "town": null,
                },
                "assignedBUCs": null,
                "contactMethods": null,
                "countryCode": "NO",
                "id": "NO:DEMO001",
                "location": null,
                "name": "DEMO001",
                "registryNumber": null,
              },
              "role": "Receiver",
              "selected": false,
            },
          ],
          "status": "empty",
          "type": "P8000",
          "version": "2",
        },
        Object {
          "allowsAttachments": true,
          "attachments": Array [],
          "creationDate": 1559117443000,
          "firstVersion": Object {
            "date": 1559117443000,
            "id": "1",
          },
          "id": "5ef5fdea78e94911a873322d98589664",
          "lastUpdate": 1559080800000,
          "lastVersion": Object {
            "date": 1559080800000,
            "id": "2",
          },
          "participants": Array [
            Object {
              "organisation": Object {
                "accessPoint": null,
                "acronym": "12348",
                "activeSince": 1518476400000,
                "address": Object {
                  "country": "NO",
                  "postalCode": null,
                  "region": null,
                  "street": null,
                  "town": null,
                },
                "assignedBUCs": null,
                "contactMethods": null,
                "countryCode": "NO",
                "id": "NO:DEMO002",
                "location": null,
                "name": "DEMO002",
                "registryNumber": null,
              },
              "role": "Sender",
              "selected": false,
            },
            Object {
              "organisation": Object {
                "accessPoint": null,
                "acronym": "12347",
                "activeSince": 1508450400000,
                "address": Object {
                  "country": "NO",
                  "postalCode": null,
                  "region": null,
                  "street": null,
                  "town": null,
                },
                "assignedBUCs": null,
                "contactMethods": null,
                "countryCode": "NO",
                "id": "NO:DEMO001",
                "location": null,
                "name": "DEMO001",
                "registryNumber": null,
              },
              "role": "Receiver",
              "selected": false,
            },
          ],
          "status": "empty",
          "type": "P7000",
          "version": "2",
        },
        Object {
          "allowsAttachments": true,
          "attachments": Array [],
          "creationDate": 1559117444000,
          "firstVersion": Object {
            "date": 1559117443000,
            "id": "1",
          },
          "id": "2cfa8cc7af2647c4ad89315a6d1b0ad7",
          "lastUpdate": 1559080800000,
          "lastVersion": Object {
            "date": 1559080800000,
            "id": "2",
          },
          "participants": Array [
            Object {
              "organisation": Object {
                "accessPoint": null,
                "acronym": "12348",
                "activeSince": 1518476400000,
                "address": Object {
                  "country": "NO",
                  "postalCode": null,
                  "region": null,
                  "street": null,
                  "town": null,
                },
                "assignedBUCs": null,
                "contactMethods": null,
                "countryCode": "NO",
                "id": "NO:DEMO002",
                "location": null,
                "name": "DEMO002",
                "registryNumber": null,
              },
              "role": "Sender",
              "selected": false,
            },
            Object {
              "organisation": Object {
                "accessPoint": null,
                "acronym": "12347",
                "activeSince": 1508450400000,
                "address": Object {
                  "country": "NO",
                  "postalCode": null,
                  "region": null,
                  "street": null,
                  "town": null,
                },
                "assignedBUCs": null,
                "contactMethods": null,
                "countryCode": "NO",
                "id": "NO:DEMO001",
                "location": null,
                "name": "DEMO001",
                "registryNumber": null,
              },
              "role": "Receiver",
              "selected": false,
            },
          ],
          "status": "empty",
          "type": "P4000",
          "version": "2",
        },
        Object {
          "allowsAttachments": true,
          "attachments": Array [],
          "creationDate": 1559117444000,
          "firstVersion": Object {
            "date": 1559117443000,
            "id": "1",
          },
          "id": "af7f8476e55f4c53b501c41fd03a25be",
          "lastUpdate": 1559080800000,
          "lastVersion": Object {
            "date": 1559080800000,
            "id": "2",
          },
          "participants": Array [
            Object {
              "organisation": Object {
                "accessPoint": null,
                "acronym": "12348",
                "activeSince": 1518476400000,
                "address": Object {
                  "country": "NO",
                  "postalCode": null,
                  "region": null,
                  "street": null,
                  "town": null,
                },
                "assignedBUCs": null,
                "contactMethods": null,
                "countryCode": "NO",
                "id": "NO:DEMO002",
                "location": null,
                "name": "DEMO002",
                "registryNumber": null,
              },
              "role": "Sender",
              "selected": false,
            },
            Object {
              "organisation": Object {
                "accessPoint": null,
                "acronym": "12347",
                "activeSince": 1508450400000,
                "address": Object {
                  "country": "NO",
                  "postalCode": null,
                  "region": null,
                  "street": null,
                  "town": null,
                },
                "assignedBUCs": null,
                "contactMethods": null,
                "countryCode": "NO",
                "id": "NO:DEMO001",
                "location": null,
                "name": "DEMO001",
                "registryNumber": null,
              },
              "role": "Receiver",
              "selected": false,
            },
          ],
          "status": "empty",
          "type": "X007",
          "version": "2",
        },
        Object {
          "allowsAttachments": true,
          "attachments": Array [],
          "creationDate": 1559117444000,
          "firstVersion": Object {
            "date": 1559117444000,
            "id": "1",
          },
          "id": "4c10c2288a23442f85b511ecfaecb0f7",
          "lastUpdate": 1559080800000,
          "lastVersion": Object {
            "date": 1559080800000,
            "id": "2",
          },
          "participants": Array [
            Object {
              "organisation": Object {
                "accessPoint": null,
                "acronym": "12348",
                "activeSince": 1518476400000,
                "address": Object {
                  "country": "NO",
                  "postalCode": null,
                  "region": null,
                  "street": null,
                  "town": null,
                },
                "assignedBUCs": null,
                "contactMethods": null,
                "countryCode": "NO",
                "id": "NO:DEMO002",
                "location": null,
                "name": "DEMO002",
                "registryNumber": null,
              },
              "role": "Sender",
              "selected": false,
            },
            Object {
              "organisation": Object {
                "accessPoint": null,
                "acronym": "12347",
                "activeSince": 1508450400000,
                "address": Object {
                  "country": "NO",
                  "postalCode": null,
                  "region": null,
                  "street": null,
                  "town": null,
                },
                "assignedBUCs": null,
                "contactMethods": null,
                "countryCode": "NO",
                "id": "NO:DEMO001",
                "location": null,
                "name": "DEMO001",
                "registryNumber": null,
              },
              "role": "Receiver",
              "selected": false,
            },
          ],
          "status": "empty",
          "type": "X005",
          "version": "2",
        },
        Object {
          "allowsAttachments": true,
          "attachments": Array [],
          "creationDate": 1559117444000,
          "firstVersion": Object {
            "date": 1559117444000,
            "id": "1",
          },
          "id": "147065531dca429a90d47b5aa3b7cf2c",
          "lastUpdate": 1559080800000,
          "lastVersion": Object {
            "date": 1559080800000,
            "id": "2",
          },
          "participants": Array [
            Object {
              "organisation": Object {
                "accessPoint": null,
                "acronym": "12348",
                "activeSince": 1518476400000,
                "address": Object {
                  "country": "NO",
                  "postalCode": null,
                  "region": null,
                  "street": null,
                  "town": null,
                },
                "assignedBUCs": null,
                "contactMethods": null,
                "countryCode": "NO",
                "id": "NO:DEMO002",
                "location": null,
                "name": "DEMO002",
                "registryNumber": null,
              },
              "role": "Sender",
              "selected": false,
            },
            Object {
              "organisation": Object {
                "accessPoint": null,
                "acronym": "12347",
                "activeSince": 1508450400000,
                "address": Object {
                  "country": "NO",
                  "postalCode": null,
                  "region": null,
                  "street": null,
                  "town": null,
                },
                "assignedBUCs": null,
                "contactMethods": null,
                "countryCode": "NO",
                "id": "NO:DEMO001",
                "location": null,
                "name": "DEMO001",
                "registryNumber": null,
              },
              "role": "Receiver",
              "selected": false,
            },
          ],
          "status": "empty",
          "type": "P10000",
          "version": "2",
        },
        Object {
          "allowsAttachments": true,
          "attachments": Array [],
          "creationDate": 1559117444000,
          "firstVersion": Object {
            "date": 1559117444000,
            "id": "1",
          },
          "id": "07cce7d3b2514676ba342caab580c573",
          "lastUpdate": 1559080800000,
          "lastVersion": Object {
            "date": 1559080800000,
            "id": "2",
          },
          "participants": Array [
            Object {
              "organisation": Object {
                "accessPoint": null,
                "acronym": "12348",
                "activeSince": 1518476400000,
                "address": Object {
                  "country": "NO",
                  "postalCode": null,
                  "region": null,
                  "street": null,
                  "town": null,
                },
                "assignedBUCs": null,
                "contactMethods": null,
                "countryCode": "NO",
                "id": "NO:DEMO002",
                "location": null,
                "name": "DEMO002",
                "registryNumber": null,
              },
              "role": "Sender",
              "selected": false,
            },
            Object {
              "organisation": Object {
                "accessPoint": null,
                "acronym": "12347",
                "activeSince": 1508450400000,
                "address": Object {
                  "country": "NO",
                  "postalCode": null,
                  "region": null,
                  "street": null,
                  "town": null,
                },
                "assignedBUCs": null,
                "contactMethods": null,
                "countryCode": "NO",
                "id": "NO:DEMO001",
                "location": null,
                "name": "DEMO001",
                "registryNumber": null,
              },
              "role": "Receiver",
              "selected": false,
            },
          ],
          "status": "empty",
          "type": "H020",
          "version": "2",
        },
        Object {
          "allowsAttachments": true,
          "attachments": Array [],
          "creationDate": 1559117444000,
          "firstVersion": Object {
            "date": 1559117444000,
            "id": "1",
          },
          "id": "7e2b28e4bc3b47e992fc06e5024bec3a",
          "lastUpdate": 1559080800000,
          "lastVersion": Object {
            "date": 1559080800000,
            "id": "2",
          },
          "participants": Array [
            Object {
              "organisation": Object {
                "accessPoint": null,
                "acronym": "12348",
                "activeSince": 1518476400000,
                "address": Object {
                  "country": "NO",
                  "postalCode": null,
                  "region": null,
                  "street": null,
                  "town": null,
                },
                "assignedBUCs": null,
                "contactMethods": null,
                "countryCode": "NO",
                "id": "NO:DEMO002",
                "location": null,
                "name": "DEMO002",
                "registryNumber": null,
              },
              "role": "Sender",
              "selected": false,
            },
            Object {
              "organisation": Object {
                "accessPoint": null,
                "acronym": "12347",
                "activeSince": 1508450400000,
                "address": Object {
                  "country": "NO",
                  "postalCode": null,
                  "region": null,
                  "street": null,
                  "town": null,
                },
                "assignedBUCs": null,
                "contactMethods": null,
                "countryCode": "NO",
                "id": "NO:DEMO001",
                "location": null,
                "name": "DEMO001",
                "registryNumber": null,
              },
              "role": "Receiver",
              "selected": false,
            },
          ],
          "status": "empty",
          "type": "H120",
          "version": "2",
        },
        Object {
          "allowsAttachments": true,
          "attachments": Array [],
          "creationDate": 1559117443000,
          "firstVersion": Object {
            "date": 1559117443000,
            "id": "1",
          },
          "id": "7c60826b53cc431f9059d8b30e4a1f7d",
          "lastUpdate": 1559080800000,
          "lastVersion": Object {
            "date": 1559080800000,
            "id": "2",
          },
          "participants": Array [
            Object {
              "organisation": Object {
                "accessPoint": null,
                "acronym": "12348",
                "activeSince": 1518476400000,
                "address": Object {
                  "country": "NO",
                  "postalCode": null,
                  "region": null,
                  "street": null,
                  "town": null,
                },
                "assignedBUCs": null,
                "contactMethods": null,
                "countryCode": "NO",
                "id": "NO:DEMO002",
                "location": null,
                "name": "DEMO002",
                "registryNumber": null,
              },
              "role": "Sender",
              "selected": true,
            },
            Object {
              "organisation": Object {
                "accessPoint": null,
                "acronym": "12347",
                "activeSince": 1508450400000,
                "address": Object {
                  "country": "NO",
                  "postalCode": null,
                  "region": null,
                  "street": null,
                  "town": null,
                },
                "assignedBUCs": null,
                "contactMethods": null,
                "countryCode": "NO",
                "id": "NO:DEMO001",
                "location": null,
                "name": "DEMO001",
                "registryNumber": null,
              },
              "role": "Receiver",
              "selected": false,
            },
          ],
          "status": "empty",
          "type": "X008",
          "version": "2",
        },
        Object {
          "allowsAttachments": true,
          "attachments": Array [],
          "creationDate": 1559117444000,
          "firstVersion": Object {
            "date": 1559117444000,
            "id": "1",
          },
          "id": "c1069862321c419e8e3ca418f7372d64",
          "lastUpdate": 1559080800000,
          "lastVersion": Object {
            "date": 1559080800000,
            "id": "2",
          },
          "participants": Array [
            Object {
              "organisation": Object {
                "accessPoint": null,
                "acronym": "12348",
                "activeSince": 1518476400000,
                "address": Object {
                  "country": "NO",
                  "postalCode": null,
                  "region": null,
                  "street": null,
                  "town": null,
                },
                "assignedBUCs": null,
                "contactMethods": null,
                "countryCode": "NO",
                "id": "NO:DEMO002",
                "location": null,
                "name": "DEMO002",
                "registryNumber": null,
              },
              "role": "Sender",
              "selected": false,
            },
            Object {
              "organisation": Object {
                "accessPoint": null,
                "acronym": "12347",
                "activeSince": 1508450400000,
                "address": Object {
                  "country": "NO",
                  "postalCode": null,
                  "region": null,
                  "street": null,
                  "town": null,
                },
                "assignedBUCs": null,
                "contactMethods": null,
                "countryCode": "NO",
                "id": "NO:DEMO001",
                "location": null,
                "name": "DEMO001",
                "registryNumber": null,
              },
              "role": "Receiver",
              "selected": false,
            },
          ],
          "status": "empty",
          "type": "H070",
          "version": "2",
        },
      ],
      "startDate": 1571731760978,
      "status": "open",
      "type": "P_BUC_02",
    }
  }
  locale="nb"
  setShowButtons={[MockFunction]}
  setValidation={[MockFunction]}
  showButtons={true}
  validation={Object {}}
>
  <div
    className="col-md-12"
  >
    <Systemtittel>
      <TypografiBase
        type="systemtittel"
      >
        <h2
          className="typo-systemtittel"
        >
          buc:step-startSEDTitle
        </h2>
      </TypografiBase>
    </Systemtittel>
    <hr />
  </div>
  <div
    className="col-md-8"
  />
  <div
    className="col-md-4"
  />
</Step2>
`;
